python/ccf/proposal_generator.py

```diff
@@ -166,7 +166,7 @@ def set_constitution(constitution_paths: List[str], **kwargs):
 
 
 @cli_proposal
-def set_js_app(bundle_path: str, **kwargs):
+def set_js_app(bundle_path: str, disable_bytecode_cache: bool = False, **kwargs):
```

Comment:
Can we have enable_bytecode_cache instead? "disable"-type flags lead to double negatives and unnecessary confusion in my opinion.

Reply:
This would change the default to not use a bytecode cache. Is that the intention?

Reply:
Would it? Why can't the default be True?

Reply:
> @letmaik thank you for adding this, could you please add a mention of it in the changelog, and ideally in the operator/JS documentation as well, with a short discussion of the memory/runtime this entails?

I extended the docs and changelog.

Reply:
I'm wary of making API decisions on the basis of what's convenient for the proposal generator CLI because our users have stopped using the generator since we've moved to JSON and the only reason we haven't axed it yet is because we haven't had the time.